src/components/BlockingViews/BlockingView.tsx

```diff
@@ -55,11 +79,15 @@ function BlockingView({
     iconHeight = variables.iconSizeSuperLarge,
     onLinkPress = () => Navigation.dismissModal(),
     shouldEmbedLinkWithSubtitle = false,
+    animationStyles = [],
+    animationWebStyle = {},
+    CustomSubtitle,
 }: BlockingViewProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
-    function renderContent() {
-        return (
+
+    const subtitleText: React.ReactElement = useMemo(
```

Comment:
Let's keep it simple here and rely on type inference:

```suggestion
    const subtitleText = useMemo(
```